xlive/H2MOD/Modules/Networking/Networking.cpp

```diff
@@ -459,14 +478,23 @@ void applyConnectionPatches()
 	{
 		DWORD addr = h2mod->Server ? addresses_dedi[i] : addresses[i];
 		WritePointer(h2mod->GetBase() + addr + 4, &unk_flt_);
+
+		// for whatever reason this code will not execute, marking GetAddress not inline or exec in debug mode fixes it
+		//WritePointer((DWORD)h2mod->GetAddress(addresses[i], addresses_dedi[i]) + 4, &unk_flt_); 
 	}
 
 	// increase max bits per second of LIVE netcode (3000 bytes -> ~8000 bytes)
-	WriteValue<DWORD>((DWORD)h2mod->GetAddress(0x1AAD63 + 6, 0x1AB268 + 6), 61440);
+	WriteValue<DWORD>((DWORD)h2mod->GetAddress(0x1AAD63, 0x1AB268) + 6, 61440);
+
+	BYTE call[] = { 0xE8 };
+	DWORD network_observer_patch_addr = (DWORD)h2mod->GetAddress(0x1BF1B9, 0x1B9093);
```

Comment:
please add a small doc here or on the method explaining why/what this does

Reply:
places call, will fix